onadata/apps/main/models/audit.py

```diff
@@ -74,9 +74,12 @@ def query_data(cls, username, query=None, fields=None, sort=None, start=0,
             or_params = []
             if '$or' in list(query):
                 or_dict = query.pop('$or')
-                for l in or_dict:
-                    or_where.extend([u"json->>%s = %s" for i in l.items()])
-                    [or_params.extend(i) for i in l.items()]
+                for or_query in or_dict:
+                    or_where.extend(
+                        [u"json->>%s = %s" for i in or_query.items()])
+                    [
```

Comment:
Codacy found an issue: [Expression "[or_params.extend(i) for i in or_query.items()]" is assigned to nothing](https://app.codacy.com/gh/onaio/onadata/pullRequest?prid=6237366)